package.json

```diff
@@ -201,6 +201,7 @@
   },
   "dependencies": {
```

Comment:
Everything under `dependencies` gets shipped with the app to the user, and takes time and space. We should keep this list minimal. We have no use for `jquery`, please remove it. I asked above about `bignumber.js`. The other 2 are great.